util/git/hooks/post-checkout

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+exec ./bazelisk.sh sync --configure
```

Comment:
I thought we might need a `--refresh` in the args to `bitstreams_workspace.py`, but it turns out that my change of the default bitstream target from "latest" to "HEAD" has inactivated that feature by default. Given that the sync is a manual operation, probably this management feature is actually not needed anyway, haha.